tests/impl.cpp

```diff
@@ -1644,11 +1942,16 @@ float compord(float a, float b)
 
     bool isNANA = isNAN(a);
     bool isNANB = isNAN(b);
-    if (!isNANA && !isNANB) {
-        ret = getNAN();
-    } else {
-        ret = 0.0f;
-    }
+    ret = (!isNANA && !isNANB) ? getNAN() : 0.0f;
+    return ret;
+}
+
+float compunord(float a, float b)
```

Comment:
Can you revise the naming? It might be confusing with "compound."

Reply:
I think we can avoid creating a new function and simply write the compare statement in `test_mm_cmpunord_ss` and `test_mm_cmpunord_ps`.